src/Tribe/Assets.php

```diff
@@ -274,26 +277,49 @@ public function enqueue( $forcibly_enqueue = null ) {
 	 * @return string|false The url to the minified version or false, if file not found.
 	 */
 	public static function maybe_get_min_file( $url ) {
-		$urls            = [];
-		$wpmu_plugin_url = set_url_scheme( WPMU_PLUGIN_URL );
-		$wp_plugin_url   = set_url_scheme( WP_PLUGIN_URL );
-		$wp_content_url  = set_url_scheme( WP_CONTENT_URL );
-		$plugins_url     = plugins_url();
+		static $wpmu_plugin_url;
```

Comment:
Any reason to not use `static $wpmu_plugin_url = set_url_scheme( WPMU_PLUGIN_URL );` for these lines, beyond that there's a bunch of them? It gets rid of the isset() conditional

Reply:
for some reason on PHP 5.6 it was complaining about that as an error.

Reply:
Ah ok, that's fine then